src/plugins/uccl/uccl_backend.cpp

```diff
@@ -0,0 +1,676 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
do you need to mention your copyright? (also relevant for other files)

Reply:
I added the copyright based on other plugins. I can change it accordingly if that is ok.